cloudbuild.yaml

```diff
@@ -56,7 +56,7 @@ steps:
   - 'CLOUD_BUILD_BRANCH_NAME=${_CLOUD_BUILD_BRANCH_NAME}'
   - 'DPU_IMAGE_NAME=${_DPU_IMAGE_NAME}'
   - 'CLOUD_BUILD_PAT_TRIGGER_ID=${_CLOUD_BUILD_PAT_TRIGGER_ID}'
-  - 'SECRET_KEY=${_SECRET_KEY}'
+  - 'NEXT_PUBLIC_GITHUB_PAT_ENCRYPTION_SECRET_KEY=${_GITHUB_PAT_ENCRYPTION_SECRET_KEY}'
```

Comment:
Low LOGAF since we are going to remove this anyway in the next PR:

But would we only use this for encryption of the GitHub PAT? Or can it be part of the encryption module itself and everything can be encrypted using the same key?

Reply:
ahh yess, the name has github because currently we are using github pat only. but yeah the secret key can be a part of the encryption module itself.